### PR TITLE
NikitaK/ Convert contact us page to TS 51832

### PR DESCRIPTION
**Pull request by NikitaK-deriv**
Changes:

- (https://redmine.deriv.cloud/issues/51832#NikitaK/Convert_contact_us_page_to_TS)

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Comment by NikitaK-deriv** [comment]
src/index.d.ts need to read svg. please comment where I should to add it

File: src/pages/contact_us/_offices.tsx [hunk 1]
```diff
@@ -182,7 +188,7 @@ const StyledMobileAddress = styled.div`
     }
 `
 
-export const Offices = () => {
+export const Offices: React.FC<GridLayoutProps> = () => {
```
Comment (prince-deriv) [inline src/pages/contact_us/_offices.tsx:191]: @NikitaK-deriv  no need to pass the `GridLayoutProps` here. It's already in the `GridLayout`  also we are moving away from `React.FC` you can just add type on the return as `ReactElement`

```
export const Offices = ():ReactElement => {
```

File: src/index.d.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+declare module '*.svg'
```
Comment (sean-binary) [inline src/index.d.ts:1]: We don't need this once you pull in the latest master.

Comment (NikitaK-deriv) [reply src/index.d.ts:1]: > We don't need this once you pull in the latest master.

Alright, got it

**Comment by sean-binary** [comment]
Please help to fix the code smell.

**Review by sean-binary — APPROVED** [review]

**Pull request merged**